askbot/jinja2/macros.html

```diff
@@ -483,7 +476,7 @@ <h2 class="comment-title">{% trans %}Comments{% endtrans %}</h2>
            class="off"
            title="{{desc_tooltip}}"><span>{{label}}</span></a>
     {% endif %}
-    <script type="text/javascript">{# need to pass on text translations to js #}
```

Comment:
This comment update was just to avoid an annoying type error on the page. It's not relevant to the change.